src/PurchaseTransactionLine.php

```diff
@@ -16,6 +18,30 @@ class PurchaseTransactionLine extends BaseTransactionLine
     use VatTotalFields;
     use ValueOpenField;
 
+    /**
+     * @var PurchaseTransaction
+     */
+    private $transaction;
+
+    /**
+     * @param PurchaseTransaction $object
+     */
+    public function setTransaction($object): void
```

Comment:
Perhaps the `setTransaction` method should throw an exception if a transaction is already set. Just to prevent weird behavior when adding a line to multiple transactions for whatever reason.